/Users/vladimirkopylov/bots/venv/bin/python3: can't open file '/Users/vladimirkopylov/bots/bots/NET_panic_bot/net_panic_bot.py': [Errno 2] No such file or directory
